src/data/products.js

```diff
@@ -0,0 +1,49 @@
+const products = [
+  {
+    id:1,
+    name:'product1',
+    price:12,
+    category:3,
+    desc:'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed eu nisl commodo, pharetra urna quis, gravida quam.' +
+    ' Maecenas sed libero tortor. Nunc commodo dolor ac vehicula ullamcorper.' +
+    ' In consectetur, ante id blandit dictum, ipsum lacus rutrum ipsum, non commodo dui libero quis nunc. ' +
+    'Nullam tincidunt leo nunc, eget lobortis justo cursus vitae. Ut mattis tristique dignissim.'
+  },
+  {
+    id:2,
+    name:'product2',
+    price:22,
+    category:2,
+    desc:'Nullam sed felis convallis, bibendum erat ut, aliquet lectus. Morbi quis nisi sit amet sem ullamcorper vehicula.' +
+    ' Duis sagittis a orci eget venenatis. In dictum metus sit amet mi egestas, ac feugiat turpis maximus.' +
+    ' Morbi posuere urna vel ex rutrum, eget sagittis odio tincidunt.' +
+    ' Mauris justo sem, fermentum sit amet consequat cursus, porta eget mauris.'
+  },
+  {
+    id:3,
+    name:'product3',
+    price:7,
+    category:1,
+    desc: 'Ut efficitur orci vel suscipit lobortis. Aenean non nisl quis dui tincidunt fermentum vel quis orci.' +
+    ' In fermentum sem ut odio iaculis, sit amet varius augue mattis.' +
+    ' Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.' +
+    ' Pellentesque et quam sit amet sapien porta bibendum.'
+  },
+  {
+    id:4,
+    name:'product4',
+    price:34,
+    category:1,
+    desc: 'Nam auctor elit risus, a maximus lectus tincidunt id. Nulla consectetur pretium auctor. Ut sit amet quam arcu. ' +
+    'Fusce dapibus posuere condimentum. Maecenas malesuada lorem magna, quis ultrices nunc laoreet sit amet. ' +
+    'Phasellus vitae erat lacus. Morbi non eros eget nibh pretium convallis tristique et lacus. '
+  },
+  {
+    id:5,
+    name:'product5',
+    price:17,
+    category:2
+  }
+]
```

Comment:
If this is defined in code I would push it further and generated products in loop. Many products, say ~500, will pose more challenge and  reveal issues we might be missing now